neural_compressor/torch/utils/auto_accelerator.py

```diff
@@ -31,6 +31,7 @@
 
 from neural_compressor.torch.utils import logger
 
+PRIORITY_HPU = 100
```

Comment:
It would be better to assign a distinct priority for each accelerator. From a business view, I believe that HPU should have a higher priority than CUDA.

Reply:
Sure, will update.